lib/cached_counts/dalli_check.rb

```diff
@@ -1,5 +1,7 @@
 if defined?(Rails)
   ActiveSupport.on_load :cached_counts do
-    raise "CachedCounts depends on Dalli!" unless Rails.cache.respond_to?(:dalli)
+    unless Rails.cache.respond_to?(:dalli) || Rails.cache.instance_variable_get(:@data).is_a?(Dalli::Client)
```

Comment:
Should this be `defined?(Dalli::Client) && Rails.cache.instance_variable_get(:@data).is_a?(Dalli::Client)`?

Reply:
@patbl Updated!